masonry/src/widget/widget_pod.rs

```diff
@@ -395,7 +395,10 @@ impl<W: Widget> WidgetPod<W> {
 
                 true
             }
-            LifeCycle::AnimFrame(_) => true,
+            LifeCycle::AnimFrame(_) => {
+                state.request_anim = false;
```

Comment:
Same here, was animation never cleared before (i.e. always requested)?

Reply:
Yeah, exactly. It would just get stuck in an infinite loop.